data/json/snippets/zombie_anatomy.json

```diff
@@ -171,6 +171,72 @@
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_electric_general>",
+    "text": [
+      "You find strange tubular glands across the creature contained alternating layers of two alien tissue types soaked in a black liquid, its smell putrid but saltier than usual for a zombie.",
+      "Uncomfortable static forces you to ground yourself multiple times during your work.",
+      "A roaming insect attracted by the carcass gets fried after landing on an organ glowing a faint blue, making you decide to avoid it with the utmost care.",
+      "The creature's heart seems to still be beating, but rather than pumping blood, it strangely reminds you of some sort of biological dynamo",
+      "The blackened veins of the creature seem strangely rigid, and you see faint crackles of electricity course across them."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_humanoid_harvest_mushroom_general>",
+    "text": [
+      "Despite the carcass lying still, you still notice churning while investigating its entrails, as if whatever was originally animating this corpse is wrestling with the mycelium fibers to retake control.",
+      "Under the stomach you notice a wrinkled fruit that you can't help but think somewhat appetizing despite where you found it.",
+      "The skin feels absolutely turgid, oozing water mixed in with traces of a black liquid where you make your incisions.",
+      "Fungal stalks still sway gently despite the lack of movement of the formerly shambling corpse.",
+      "The putrefied blood that leaks from your incisions is visibly saturated with spores."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_animal_harvest_mushroom_general>",
+    "text": [
+      "Despite the carcass lying still, you still notice churning while investigating its entrails, as if whatever was originally animating this corpse is wrestling with the mycelium fibers to retake control.",
+      "Under the stomach you notice a wrinkled fruit that you can't help but think somewhat appetizing despite where you found it.",
+      "The skin feels absolutely turgid, oozing water mixed in with traces of a black liquid where you make your incisions.",
+      "Fungal stalks still sway gently despite the lack of movement of the formerly shambling corpse.",
+      "The putrefied blood that leaks from your incisions is visibly saturated with spores."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_pupating_general>",
+    "text": [
+      "Aside from the obvious large cavity that used to house its spawn, it seems this undead monstrosity had another, smaller cavity within its skull which displaced most of its brain in which you find a small malformed corpse of...  something.",
+      "Judging from the various half eaten copies of its main burden, it seems the inside of this creature were used as a makeshift arena from which only the strongest could triumph.",
```

Comment:
These three feel a bit too prescriptive - deciding for the reader that it's a "makeshift arena" / "industrial production line"/ to a lesser extent "discarded by something greater" instead of letting them come to this conclusion themselves. Ex
`Every cavity of this creature is filled with smaller offspring, some still in the process of devouring each other / The corpse is inundated with smaller forms, increasing in complexity and size as you near its central cavity where they wait for their release in neat clusters` for the first two, though "the kids eating each other" feels wasteful in a more mutant-y way than Blobby.